lib/general.sh

```diff
@@ -1338,31 +1327,34 @@ prepare_host()
 	# packages list for host
 	# NOTE: please sync any changes here with the Dockerfile and Vagrantfile
 
-# build aarch64
+	local hostdeps="acl aptly aria2 bc binfmt-support bison btrfs-progs       \
```

Comment:
Nitpick: I'd use bash arrays (`declare -a`) for `hostdeps`. 
array syntax allows for multiline without needing `\` so easier to reformat later.

Reply:
Installing packages using the install_pkg_deb function, which requires a list separated by a space, tab, or a newline character.

Verification code:
```
#!/bin/bash

	      hostdeps="acl aptly aria2 bc binfmt-support bison btrfs-progs       \
	build-essential  ca-certificates ccache cpio cryptsetup curl              \
	debian-archive-keyring debian-keyring debootstrap device-tree-compiler    \
	dialog dirmngr dosfstools dwarves f2fs-tools fakeroot flex gawk           \
	gcc-arm-linux-gnueabihf gdisk gnupg1 gpg imagemagick jq kmod libbison-dev \
	libc6-dev-armhf-cross libelf-dev libfdt-dev libfile-fcntllock-perl        \
	libfl-dev liblz4-tool libncurses-dev libpython2.7-dev libssl-dev          \
	libusb-1.0-0-dev linux-base locales lzop ncurses-base ncurses-term        \
	nfs-kernel-server ntpdate p7zip-full parted patchutils pigz pixz          \
	pkg-config pv python3-dev python3-distutils qemu-user-static rsync swig   \
	systemd-container u-boot-tools udev unzip uuid-dev wget whiptail zip      \
	zlib1g-dev lib32ncurses-dev"

	list_deps="
	acl aptly aria2 bc binfmt-support bison btrfs-progs
	build-essential  ca-certificates ccache cpio cryptsetup curl
	debian-archive-keyring debian-keyring debootstrap device-tree-compiler
	dialog dirmngr dosfstools dwarves f2fs-tools fakeroot flex gawk
	gcc-arm-linux-gnueabihf gdisk gnupg1 gpg imagemagick jq kmod libbison-dev
	libc6-dev-armhf-cross libelf-dev libfdt-dev libfile-fcntllock-perl
	libfl-dev liblz4-tool libncurses-dev libpython2.7-dev libssl-dev
	libusb-1.0-0-dev linux-base locales lzop ncurses-base ncurses-term
	nfs-kernel-server ntpdate p7zip-full parted patchutils pigz pixz
	pkg-config pv python3-dev python3-distutils qemu-user-static rsync swig
	systemd-container u-boot-tools udev unzip uuid-dev wget whiptail zip
	zlib1g-dev lib32ncurses-dev
	"


test_print ()
{
	echo "$*"
	echo ""
	echo -e "$*"
	echo -e "\n=======\n"
}

test_print $hostdeps

test_print $list_deps
```

In this case, I want to show that if you want to write a long line,
then it should be separated by the "`\`" character at the 79th position of the line.
I.e., the line in script should not be longer than 80.

It is very difficult to read and correct the text of the code when you look at it through
a small screen panel in a console text editor. For example, directly on a working board.

Reply:
Sorry. That's _definitely_ not what I meant. Sorry.
I meant:
```bash
declare -a list_of_things=("one-thing" "another-thing"
                           "this-does-not-need-backslashes")
[[ a == a ]] && list_of_things+=("add_one_thing_later")
echo "${list_of_things[@]}" # expand the array correctly into a string, each thing a parameter
echo "${list_of_things[*]}" # expand into a single parameter
```

Reply:
Do I understand correctly that you need a list of listed packages designed as an array but not as a string?
Is it easier for you to work with an array?